src/standalone-mocks/frontend-components-mock.tsx

```
@@ -1,50 +1,25 @@
 import React from 'react';
 
-export const PageHeader: React.FC<Record<string, unknown>> = ({
-  children,
-  ...props
-}): JSX.Element => {
+export const PageHeader: React.FC<any> = ({ children, ...props }) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)